<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="generalTab">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QGroupBox" name="ownCloudGroupBox">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="title">
            <string>Your ownCloudServer</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <widget class="QLineEdit" name="serverNameEdit">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>50</y>
              <width>291</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>121</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>username:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>121</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>https://</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="userNameEdit">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>90</y>
              <width>291</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>130</y>
              <width>121</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>password:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="passwordEdit">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>130</y>
              <width>291</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QPushButton" name="connectButton">
            <property name="geometry">
             <rect>
              <x>450</x>
              <y>50</y>
              <width>161</width>
              <height>111</height>
             </rect>
            </property>
            <property name="text">
             <string>connect</string>
            </property>
            <property name="icon">
             <iconset theme="network-wired"/>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
